ddp-workspace/projects/ddp-atcp/src/app/components/data-access/data-access.html

```diff
@@ -0,0 +1,361 @@
+<div class="wizard-page">
+  <app-header></app-header>
+  <div class="wizard">
+    <app-progress-bar [count]="countTabs" [active]="activeTab" (onChanged)="setActiveTab($event)"></app-progress-bar>
+  </div>
+  <form class="content" #form="ngForm" (ngSubmit)="submit(form)" [ngClass]="{submitted : form.submitted}">
+    <div [ngClass]="{'active' : activeTab === 0}">
+      <h2 translate>DataAccess.Header.Title</h2>
+      <p translate>DataAccess.Header.Paragraph1</p>
+      <p translate>DataAccess.Header.Paragraph2</p>
+      <p translate>DataAccess.Header.Paragraph3</p>
+      <p translate>DataAccess.Header.Paragraph4</p>
+      <p translate>DataAccess.Header.Paragraph5</p>
+      <hr>
+    </div>
+    <div [ngClass]="{'active' : activeTab === 1 }">
+      <h2 translate>DataAccess.ResearcherForm.Title</h2>
+      <div>
+        <div class="form-group">
+          <label for="name" translate>DataAccess.ResearcherForm.Name</label>
+          <input type="text"
+                 class="form-control"
+                 id="name"
+                 required
+                 name="data_access_request[researcher_name]"
+                 [(ngModel)]="name">
+          <span class="error" *ngIf="form.submitted && !name">
+            Researcher name cannot be empty.
```

Comment:
Please, move the text in the translation file and review the entire file, there are other examples of English text.